### PR TITLE
added gettempemail.com helper

### PR DESCRIPTION
**Pull request by gettempemaildev**
# Thank you for opening a pull request to tiny-helpers.dev ❤️

Before you submit your PR, please take a look at the [contributing guidelines](https://github.com/stefanjudis/tiny-helpers/blob/master/CONTRIBUTING.md). :)

To make it easier for us to review and merge your PR, please make sure …

- [x] you only add one (!) new helper per pull request.
- [x] you have checked if an open PR already exists.
- [x] the submitted website is focused on a single, development related issue.
- [x] the `desc` field includes an "actionable sentence" (e.g. "Create something great" or "Transform something into something else").

Thank you! 🙇‍♂️

### CODE REVIEW

File: helpers/gettempemail.com.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "gettempemail.com",
+  "desc": "create disposable email within seconds",
```
Comment (stefanjudis) [inline helpers/gettempemail.com.json:3]:
```suggestion
  "desc": "Create disposable email addresses within seconds",
```

File: helpers/gettempemail.com.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "url": "gettempemail.com",
+  "tags": [
+    "Email",
+    "Misc",
+    "Security"
+  ],
+  "maintainers": [
+    "shivamethicalhat",
+    "gettempemaildev"
```
Comment (stefanjudis) [inline helpers/gettempemail.com.json:12]: This looks like a generic product account. Could we remove it?

**Comment by stefanjudis** [comment]
This is great. :) Thanks so much for contributing. I left two comments. :)

File: helpers/gettempemail.com.json (hunk 2 above)
Comment (gettempemaildev) [reply helpers/gettempemail.com.json:12]: sure. both?

Comment (gettempemaildev) [reply helpers/gettempemail.com.json]: removed both  :-)

**Comment by stefanjudis** [comment]
I'll add the first one back. :) Individual maintainers are great I just want to avoid generic or company accounts. :) 

Thanks a bunch for contributing!

**Pull request merged**